packages/office-ui-fabric-react/src/components/Dialog/Dialog.Props.ts

```diff
@@ -9,126 +10,103 @@ export interface IDialog {
 
 }
 
-export interface IDialogContentProps extends React.Props<DialogContent>, IWithResponsiveModeState, IAccessiblePopupProps {
-  /**
-  * Show an 'x' close button in the upper-right corner
-  */
-  showCloseButton?: boolean;
+export interface IDialogProps extends React.Props<Dialog>, IWithResponsiveModeState, IAccessiblePopupProps {
 
   /**
-   * Other top buttons that will show up next to the close button
+   * Optional callback to access the IDialog interface. Use this instead of ref for accessing
+   * the public methods and properties of the component.
    */
-  topButtonsProps?: IButtonProps[];
-
-  /**
-  * Optional override class name
-  */
-  className?: string;
+  componentRef?: (component: IDialog) => void;
 
   /**
-  * A callback function for when the Dialog is dismissed from the close button or light dismiss, before the animation completes.
+  * Props to be passed through to Dialog Content
   */
-  onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
+  contentProps?: IDialogContentProps;
 
   /**
-  * The subtext to display in the dialog.
+  * Props to be passed through to Modal
   */
-  subText?: string;
+  modalProps?: IModalProps;
 
   /**
-  * The title text to display at the top of the dialog.
+  * A callback function for when the Dialog is dismissed from the close button or light dismiss. Can also be specified separately in content and modal.
   */
-  title?: string;
-}
-
-export interface IDialogProps extends React.Props<Dialog>, IWithResponsiveModeState, IAccessiblePopupProps {
-  /**
-   * Optional callback to access the IDialog interface. Use this instead of ref for accessing
-   * the public methods and properties of the component.
-   */
-  componentRef?: (component: IDialog) => void;
+  onDismiss?: (ev?: React.MouseEvent<HTMLButtonElement>) => any;
 
   /**
   * Whether the dialog is displayed.
   * @default false
+  * @deprecated Pass through via dialogModalProps instead
```

Comment:
I'm confused. You added `modalProps` but all of your comments say `dialogModalProps`.

The `isOpen` prop is rather important, and also poorly done.

The default behavior of a dialog, without props, is that it's *visible*, not hidden. Otherwise, a customer who tries out Dialog is very confused when they don't provide isOpen=true.

I would really like isOpen to be deprecated, in favor of something like `hidden`, which has a default of `false`, but by the comments alone, I'm really confused.